renovate.json5

```diff
@@ -30,13 +30,13 @@
     // Group auxiliary angular-related updates
     {
       "matchDepPatterns": ["^angular"],
-      "excludePackagePatterns": ["^@xlts.dev/"],
+      "excludePackagePatterns": ["^@neverendingsupport/"],
       "groupName": "auxiliary angular packages"
     },
-    // Group xlts angular-related updates
+    // Group AngularJS Never-Ending Support updates (from HeroDevs)
     {
       "matchSourceUrlPrefixes": ["https://github.com/xlts-dev/"],
```

Comment:
I don't think this will work since the URL changed.

Reply:
I am not sure where this originally came from. I cannot find this URL in the current packages.

Let's try it and see if Renovate can figure it out.